.gitignore

```diff
@@ -1,3 +1,4 @@
+.idea/
 # Byte-compiled / optimized / DLL files
```

Comment:
General: Consider spliting the files to sub directories (for instance: Utils, Entities, etc...)